extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -103,6 +107,19 @@ export class AmpSidebar extends AMP.BaseElement {
     this.registerAction('toggle', this.toggle_.bind(this));
     this.registerAction('open', this.open_.bind(this));
     this.registerAction('close', this.close_.bind(this));
+
+    installGlobalClickListenerForDoc(this.getAmpDoc()).addBeforeHandler(e => {
```

Comment:
Just checked you're right it's not capture. I could swear it was 😅 when I checked